_layouts/page.html

```diff
@@ -3,7 +3,7 @@
 ---
 
 <!-- Page Header -->
-<header class="intro-header" style="background-image: url('{{ site.baseurl }}/{% if page.header-img %}{{ page.header-img }}{% else %}{{ site.header-img }}{% endif %}')">
+<header class="intro-header" style="background-image: url('http://cdn.shopify.com/s/files/1/0051/4802/products/Decal_1024x1024.png?v=1359081115')">
```

Comment:
@brianamarie, the image is great, but... 

- Are we allowed to use this image? Who legally owns this image?
- I know it's a CDN path, but we shouldn't serve images from external resources.